msu/systems/mod_settings/settings_page.nut

```diff
@@ -0,0 +1,105 @@
+::MSU.Class.SettingsPage <- class
+{
+	Name = null;
+	ID = null;
+	Settings = null;
+	Parent = null;
```

Comment:
Parent should be renamed to Panel or something that immediately tells us what this variable is. The associated setParent and similar functions should be renamed to setPanel etc.